### DISCUSSION TITLE
Transmog Item Sorting

### DISCUSSION THREAD

**Discussion by Peti321** [chromiecraft/chromiecraft#7316] [Suggestions]
Firstly let me say that I love the new transmog UI changes.
However I miss the "Search" button from the old one.
I know it is kinda a "first world problem" but I have a lot of mogs and I dont see a way to sort it anyway.
My suggestions:
-Have the search function somehow back
-Make the items alphabetical
-Make item sorted automatically as their rarity (so greens first, then blues, then purples)
Thank you!

**Comment by Helias** [comment]
Hi, thanks for the suggestions, we'll see  how we could improve the search and sort features.

**Comment by Helias** [comment]
Now the items are sorted by quality and by name, thanks for the suggestion!

**Reply by Peti321** [reply]
It's awesome, thank you! :D :)
